GOT-StudentVersion/GOT-StudentVersion/Controllers/GOTETableViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  GOTETableViewController.swift
+//  GOTE Project
+//
+//  Created by Mariel Hoepelman on 8/18/19.
+//  Copyright © 2019 Mariel Hoepelman. All rights reserved.
+//
+
+import UIKit
+
+class GOTETableViewController: UITableViewController {
+    var episode = GOTEpisode.allEpisodes
+    
```

Comment:
You're missing an instance of the model in this format to avoid hard coding like below:
    var arrOfSeasons = [[GOTEpisode]]() {
        didSet{
            tableView.reloadData()
        }
    }
    
    private func filterSeason () -> [[GOTEpisode]] {
        arrOfSeasons = [GOTEpisode.allEpisodes.filter{$0.season == 1}, GOTEpisode.allEpisodes.filter{$0.season == 2},GOTEpisode.allEpisodes.filter{$0.season == 3},GOTEpisode.allEpisodes.filter{$0.season == 4},GOTEpisode.allEpisodes.filter{$0.season == 5},GOTEpisode.allEpisodes.filter{$0.season == 6}, GOTEpisode.allEpisodes.filter{$0.season == 7}]
        return arrOfSeasons
    }

Reply:
So not that matrix holds the value of and all the episodes and seasons sored by season